spec/javascript-spec.coffee

```diff
@@ -1220,11 +1224,6 @@ describe "Javascript grammar", ->
       expect(tokens[3]).toEqual value: '.', scopes: ['source.js', 'meta.delimiter.property.period.js']
       expect(tokens[4]).toEqual value: 'b', scopes: ['source.js', 'variable.other.property.js']
 
-      {tokens} = grammar.tokenizeLine('a.123illegal')
```

Comment:
Is this impossible to test for now?  Can't we just give numbers higher priority?